scalableminds/webknossos
# Add histogram to dataset settings

This PR adds the long-awaited histogram feature for to the dataset settings. It uses the new histogram backend route to get the sampled histogram data of the current dataset.

### URL of deployed dev instance (used for testing):
- [https://addhistogramtodatasetsettings.webknossos.xyz](https://addhistogramtodatasetsettings.webknossos.xyz)

### Steps to test:
- Open some tracing
- There open the dataset settings
- Each layer should have its own histogram. With this, you can adjust the max and min value.

### Issues:
- fixes #3583

### ToDo:
- [x] Discuss what to do the [legacy options](https://github.com/scalableminds/webknossos/blob/586a7f4cd83ecc9b1456fc1670cbf7ce56d259b7/frontend/javascripts/libs/datasource.schema.json#L143) of brightness and contrast. 

------
